bucket.go

```diff
@@ -20,6 +20,8 @@ type Bucket interface {
 	GetName() string
 	Get(k string, rv interface{}) error
 	GetRaw(k string) ([]byte, error)
+	Gets(k string, rv interface{}, caso *uint64) error
```

Comment:
>  just replace our Get(key, interface{}) error with Get(key, interface{}) (cas, error)

Yeah that seems cleaner to me.  Closer to go-cb's approach:

https://godoc.org/github.com/couchbaselabs/gocb#Bucket.Get